medicai/transforms/random/random_crop_pos_neg.py

```
@@ -86,126 +103,218 @@ def __init__(
         pos: int,
         neg: int,
         num_samples: int = 1,
+        *,
+        input_layout: str,
         image_reference_key: str = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)